src/libraries/mysterium-client/launch-daemon/launch-daemon-installer.js

```diff
@@ -66,6 +66,7 @@ class LaunchDaemonInstaller implements Installer {
             <string>${this._config.openVPNBin}</string>
             <string>--tequilapi.port</string>
             <string>${this._config.tequilapiPort}</string>
+            <string>--experiment-promise-check=1</string>
```

Comment:
?? Key-value pairs seem to be separated in this config, we should keep it consistant if possible - is that possible for this case?
```suggestion
            <string>--experiment-promise-check</string>
            <string>1</string>
```

Reply:
unfortunately this doesn't work ;/